pkg/volume/quobyte/quobyte_util.go

```diff
@@ -46,3 +93,53 @@ func (mounter *quobyteMounter) pluginDirIsMounted(pluginDir string) (bool, error
 func (mounter *quobyteMounter) correctTraillingSlash(regStr string) string {
 	return path.Clean(regStr) + "/"
 }
+
+func validateRegistry(registry string) bool {
+	if len(registry) == 0 {
+		return false
+	}
+
+	for _, hostPortPair := range strings.Split(registry, ",") {
+		if _, _, err := net.SplitHostPort(hostPortPair); err != nil {
+			return false
+		}
+	}
+
+	return true
+}
+
+// ADD Annotations to a PersistentVolume all information needed to delete the Volume
+func addVolumeAnnotations(apiServer, adminSecret, adminSecretNamespace string, pv *api.PersistentVolume) {
```

Comment:
Yes we do have a common case of using pv annotation. Please refactor as needed. Thanks